code/frameworks/nextjs/src/dependency-map.ts

```diff
@@ -6,17 +6,17 @@ const mapping: Record<string, Record<string, string>> = {
   '<11.1.0': {
     'next/dist/next-server/lib/router-context': 'next/dist/next-server/lib/router-context',
   },
-  '>=11.1.0': {
+  '>=11.1.0 <13.5.0': {
     'next/dist/shared/lib/router-context': 'next/dist/shared/lib/router-context',
   },
-  '>=13.5.0': {
-    'next/dist/shared/lib/router-context': 'next/dist/shared/lib/router-context.shared-runtime',
-    'next/dist/shared/lib/head-manager-context':
-      'next/dist/shared/lib/head-manager-context.shared-runtime',
-    'next/dist/shared/lib/app-router-context':
-      'next/dist/shared/lib/app-router-context.shared-runtime',
-    'next/dist/shared/lib/hooks-client-context':
-      'next/dist/shared/lib/hooks-client-context.shared-runtime',
+  '< 13.5.0': {
```

Comment:
Will this ever be hit? given that the `>=11.1.0 <13.5.0` also includes `<13.5.0`. Also, there's a space there, I don't know if it matters.

```suggestion
  '<13.5.0': {
```